civic_sandbox/urls.py

```diff
@@ -0,0 +1,14 @@
+from django.conf.urls import url
+from rest_framework.urlpatterns import format_suffix_patterns
+from . import views
+
+urlpatterns = [
+    url(r'^bikeparking/', views.bikeparking),
+    url(r'^bikelanes/', views.bikelanes),
+    url(r'^taxlotblockgroups/(?P<date_filter>\d+)', views.taxlotblockgroups),
```

Comment:
How does this `date_filter` thing work? In the absence of unit tests, can you provide an example URL that this captures?

Reply:
Since we are not building in additional querying for sandbox endpoints beyond the date filter, this will just add /year or /monthyear or /seasonyear to the url. Example: http://localhost:8000/neighborhood-development/sandbox/taxlotblockgroups/2011/

Do you want me to document this somewhere? If so, where?

Reply:
If everyone is cool with that format, that's fine. I just expected query params.

Reply:
Has front-end or anyone specified a preferred format on encoding inputs in query params vs. URL structure? Seems like using query params would be more explicit, in a self-documenting manner. Also keeps the main part of the URL the same.

If the existing use of encoding via URL structure is kept, adding a unit test case in [tests.py](https://github.com/hackoregon/neighborhoods-2018/blob/civic_sandbox/civic_sandbox/tests.py) would not only document the format, but also verify that it works correctly. (Maybe test each of the three inputs: i.e. /year, /monthyear, and /seasonyear)